### PR TITLE
switch challenges solution

### PR DESCRIPTION
**Pull request by rocheamor**
hi, kindly review my code. thanks

### CODE REVIEW

File: switchStatements/switch-challenge.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+let parent2 = "green";
+let eyeCombo = parent1 + parent2;
+let childEyeColor;
+
+switch (eyeCombo){
+    case "brownbrown":
+        console.log(`Your eye colours are ${parent1} and ${parent2}, the chances of your child having blue eyes is 6.25%, brown is 75% and green is 18.75%`);
+        childEyeColor = "brown";
+        break;
+    case "greenbrown":
```
Comment (kaizdev) [inline switchStatements/switch-challenge.js:56]: cool use of switch statements

**Review by sunny9312-lab — APPROVED** [review]
that looks good

**Review by kaizdev — APPROVED** [review]

File: switchStatements/switch-challenge.js (hunk 1 above)
Comment (rocheamor) [reply switchStatements/switch-challenge.js:56]: thanks, i just though it would be simpler this way

Comment (sunny9312-lab) [reply switchStatements/switch-challenge.js:56]: you used this switch structure well.

**Pull request merged**